README.md

```diff
@@ -0,0 +1,47 @@
+# Desert.vim
+
+I started with the evening color scheme included in the distro, and tweaked a
+few colors to my liking.  Then I decided to do a major overhaul and give the
+whole thing a desert evening sort of feel.  I find it a very pleasing,
+easy-on-the-eyes theme.
+
+<a href="http://hans.fugal.net/vim/colors/desert.html">
```

Comment:
Can't you just do

  ![Screenshot](http://hans.fugal.net/vim/colors/desert.html)